doofinder.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
Remove these line breaks that you have added because they make the Prestashop validator crash.

Reply:
Sorry, they were added automatically by the editor, maybe due to some misconfiguration with `.editorconfig`
I'll remove them.
Thanks.